charts/kargo/templates/controller/deployment.yaml

```diff
@@ -44,18 +44,32 @@ spec:
         envFrom:
         - configMapRef:
             name: kargo-controller
-        {{- if or .Values.kubeconfigSecrets.kargo .Values.kubeconfigSecrets.argocd }}
+        {{- if or .Values.kubeconfigSecrets.kargo .Values.kubeconfigSecrets.argocd .Values.controller.committer.signingKeySecret.name }}
         volumeMounts:
+        {{- if .Values.controller.committer.signingKeySecret.name }}
+        - mountPath: /etc/kargo/committer
+          name: committer
+          readOnly: true
+        {{- end }}
+        {{- if or .Values.kubeconfigSecrets.kargo .Values.kubeconfigSecrets.argocd }}
         - mountPath: /etc/kargo/kubeconfigs
           name: kubeconfigs
           readOnly: true
         {{- end }}
+        {{- end }}
         securityContext:
           {{- toYaml .Values.controller.securityContext | nindent 10 }}
         resources:
           {{- toYaml .Values.controller.resources | nindent 10 }}
-      {{- if or .Values.kubeconfigSecrets.kargo .Values.kubeconfigSecrets.argocd }}
+      {{- if or .Values.kubeconfigSecrets.kargo .Values.kubeconfigSecrets.argocd .Values.controller.committer.signingKeySecret.name }}
       volumes:
+      {{- if .Values.controller.committer.signingKeySecret.name }}
+      - name: committer
+        secret:
+          secretName: {{ .Values.controller.committer.signingKeySecret.name }}
+          defaultMode: 0644
+      {{- end }}
```

Comment:
Similar nit as [here](https://github.com/akuity/kargo/pull/1843/files#r1570916323)

Reply:
Updated

[refactor(charts): order volumes and mounts more logically](https://github.com/akuity/kargo/pull/1843/commits/5e41a69a00a3ac71e2f51afe7c201dc49119b7c8)